src/library/Uno.Material/Styles/Controls/Flyout.xaml

```diff
@@ -28,6 +28,46 @@
 	<!-- Material FlyoutPresenter Style -->
 	<Style x:Key="MaterialFlyoutPresenterStyle"
 		   TargetType="FlyoutPresenter">
+		<Setter Property="HorizontalAlignment"
+				Value="Stretch" />
+		<Setter Property="VerticalAlignment"
+				Value="Stretch" />
+		<Setter Property="HorizontalContentAlignment"
+				Value="Stretch" />
+		<Setter Property="VerticalContentAlignment"
+				Value="Stretch" />
+		<Setter Property="MinWidth"
+				Value="NaN" />
+		<Setter Property="MinHeight"
+				Value="NaN" />
+		<Setter Property="MaxWidth"
+				Value="NaN" />
+		<Setter Property="MaxHeight"
+				Value="NaN" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="FlyoutPresenter">
+					<Border HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+							VerticalAlignment="{TemplateBinding VerticalAlignment}">
+						<Border.Transitions>
+							<TransitionCollection>
+								<PaneThemeTransition Edge="Bottom" />
+							</TransitionCollection>
+						</Border.Transitions>
+						<ContentPresenter Content="{TemplateBinding Content}"
+										  ContentTemplate="{TemplateBinding ContentTemplate}"
+										  ContentTransitions="{TemplateBinding ContentTransitions}"
+										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}" />
+					</Border>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<!-- Material Content FlyoutPresenter Style -->
+	<Style x:Key="MaterialContentFlyoutPresenterStyle"
```

Comment:
what's the intended use of this style? Do we need to keep it?

Reply:
Yes @davidjohnoliver, it's to have a proper flyout style for buttons for example with corner radius and content inside.